### PR TITLE
Fixing #1191 by changing to event.get method

### PR DESCRIPTION
**Pull request by andrewthetechie**
This should fix https://github.com/errbotio/errbot/issues/1191

After this change, sending an admin warning to DnD'd slack admins does this:

[dnd_slack_admin.txt](https://github.com/errbotio/errbot/files/1958305/dnd_slack_admin.txt)

### CODE REVIEW

File: errbot/backends/slack.py [hunk 1]
```diff
@@ -504,7 +504,7 @@ def _message_event_handler(self, event):
             if subtype == "bot_message":
                 msg.frm = SlackBot(
                     self.sc,
-                    bot_id=event['bot_id'],
+                    bot_id=event.get('bot_id'),
```
Comment (zoni) [inline errbot/backends/slack.py]: I'm a little bit worried what kind of effects this change could have where `bot_id` ends up being `None` in cases, but let's see how it goes. :smile: 

That said, if this is changed here, the same should be applied to the initialization of `SlackRoomBot` a little further down (line 519 currently).

Comment (andrewthetechie) [reply errbot/backends/slack.py]: Made that change.

**Comment by GenPage** [comment]
@andrewthetechie @zoni I recommend reviewing #1073 as it seems these are both similar.

**Comment by gbin** [comment]
I merged #1073

**Comment by gbin** [comment]
Thx

**Comment by zoni** [comment]
I think I prefer this approach over the one inf #1073 though. Returning `None` seems like a better API design over returning empty strings when these values can't be found.

@gbin what're your thoughts on that?

**Comment by gbin** [comment]
@zoni you have a good point, I assumed the empty string was there to no break something else.

**Comment by zoni** [comment]
From my understanding, the immediate problem is a KeyError exception on trying to access values that don't exist in the slack event. I suspect if it breaks further down the line, it'll currently break whether they're empty strings or None values.

**Comment by gbin** [comment]
directly patched on master. We'll see where it goes.

**Pull request closed without merging**